### ISSUE TITLE
Licensing Training Framework

### ISSUE THREAD

**Issue by rebotnix** [YuanxunLu/LiveSpeechPortraits#33]
Hi, 

i didnt see the training part in this repo.

Is there a way to license the training code or framework?

Thx.

**Comment by YuanxunLu** [comment]
Thanks for your attention on our work! This work was performed during my internship at the company, and I'm not allowed to release the training codes due to the company copyright issues:( - some key parts of the training codes were built based on the company's private framework. I've detailed illustrated the training details in the paper (see sec. 4) and the released testing codes could help for the training re-implementation (see loss/dataset/models.py etc.). 

If you meet troubles during the re-implementation, feel free to open an issue on the github.

**Comment by cucdengjunli** [comment]
same question
